public/src/components/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Axios from 'axios';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recipe: []
+    };
+    this.getRecipe = this.getRecipe.bind(this)
+  };
+  getRecipe() {
+    Axios.get('http://127.0.0.1:3100/recipe')
```

Comment:
your route should be more specific than just /recipe; should be /recipe/:id/instructions or something similar